### ISSUE TITLE
sub project adding dependencies

### ISSUE THREAD

**Issue by ghost** [microsoft/rushstack#3782]
pnpm  i dayjs -r --filter my-toolchain

Does Rush support the fast -to -child project to add dependencies?

**Comment by iclanton** [comment]
No, `rush add` currently doesn't support the project selector arguments (like `--to` and `--from`), but it may be straightforward to add support for them.

**Comment by Mr-GaoYu** [comment]
> No, `rush add` currently doesn't support the project selector arguments (like `--to` and `--from`), but it may be straightforward to add support for them.

Does it support the extension of the built-in command of Rush
